docs/tutorial/kymographs.rst

```diff
@@ -5,37 +5,65 @@ Kymographs
 
     :nbexport:`Download this page as a Jupyter notebook <self>`
 
-To load an HDF5 file and lists all of the kymographs inside of it, run::
+We can use the :attr:`lk.File.kymos <lumicks.pylake.File.kymos>` attribute to access the kymographs from a file::
 
     import lumicks.pylake as lk
 
-    file = lk.File("example.h5")
-    list(file.kymos)  # e.g. shows: "['cas9', 'reference']"
+    file = lk.File("kymo.h5")
+    print(file.kymos)  # dict of available kymos: {'16': Kymo(pixels=699)}
 
-Once again, :attr:`.kymos <lumicks.pylake.File.kymos>` is a regular Python dictionary so we can
-easily iterate over it::
+This is a regular Python dictionary so we can easily iterate over it::
 
-    # Plot all kymos in a file
-    >>> for name, kymo in file.kymos.items():
-            print(f"{name}, starts at {kymo.start} ns")
-    cas9, starts at 1586776312560250200 ns
-    reference, starts at 1586777007674285402 ns
+    # print some details for all kymos in a file
+    for name, kymo in file.kymos.items():
+        print(f"kymograph '{name}', starts at {kymo.start} ns")
 
-Or just pick a single one::
+    # kymograph '16', starts at 1638534513847557200 ns
 
-    kymo = file.kymos["cas9"]
-    kymo.plot(channel="green", aspect="auto", adjustment=lk.ColorAdjustment(0, 5))
+Or access a particular `Kymo` object directly to work with::
+
+    # use the kymo name as a dict key
+    kymo = file.kymos["16"]
+
+    # or get the first kymo from the dict
+    kymo_name, kymo = file.kymos.popitem()
+
+Plotting and exporting
+----------------------
+
+We can use the :meth:`~lumicks.pylake.kymo.Kymo.plot()` convenience function to plot individual color channels and the full RGB image::
+
+    plt.figure()
+
+    # plot just the green channel
+    plt.subplot(2, 1, 1)
+    kymo.plot("green", adjustment=lk.ColorAdjustment(0, 15, mode="absolute"))
+
+    # plot full color
+    plt.subplot(2, 1, 2)
+    kymo.plot(channel="rgb", adjustment=lk.ColorAdjustment(0, 99, mode="percentile"))
+
+    plt.show()
 
 .. image:: figures/kymographs/kymo_intro.png
 
-Here we see the :meth:`~lumicks.pylake.kymo.Kymo.plot()` convenience function. The `channel`
-argument accepts the strings "red", "green", "blue", or "rgb". This function accepts keyword
-arguments that are passed to :func:`plt.imshow() <matplotlib.pyplot.imshow()>` internally. Note
-also, the axes are labeled with the appropriate time and position units.
+Note that the axes are labeled with the appropriate time and position units.
+
+The first argument `channel` accepts the strings "red", "green", "blue", or "rgb". We also see that the color limits can be set easily using
+the :class:`~lumicks.pylake.ColorAdjustment` class. The first two arguments act like the `vmin` and `vmax` arguments used with
```

Comment:
Might be nice to mention that these can also be passed per color? Then again, it's already mentioned in the API docs. Maybe superfluous.

Reply:
yeah I think leave it for the API docs is fine